services/configuration_repository_service.rb

```diff
@@ -118,7 +121,6 @@ def create_remote_json_file(file_path, json_string: "[]")
     # @return [Boolean]
     def remote_file_a_json_array?(file_path)
       return false unless configuration_repository_exists?
```

Comment:
You don't have to fix anything here, but I like having an empty line after precondition checks right at the start of methods. I think of these things as `guard` statements. They should be separated, from the actual business logic in my mind.

Reply:
I think @AndrewMcBurney is on this already, cc him to let him take note of it 😄